docassemble/ALToolbox/ThreePartsDate.py

```diff
@@ -227,7 +223,7 @@
 
     // Convert the month number to a month name for the option text
     var date = new Date(1970, month, 1);
-    $opt.text(date.toLocaleString('default', {{ month: 'long' }}));
+    $opt.text(date.toLocaleString(document.documentElement.lang || 'default', {{ month: 'long' }}));
```

Comment:
I like the idea, but I have some concerns. I may be off base here, but from what I've read so far this only works for getting the `<html lang=foo>` value. I believe that would need to be set programmatically by us. It may not have any impact at all, but I'm concerned that if we don't set that value properly ourselves this may interfere with the other ways that da or the browser can set or translate the page. Do you know anything more beyond that?